bears/css/StyleLintBear.py

```diff
@@ -0,0 +1,32 @@
+import os
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='stylelint',
+        output_format='regex',
+        output_regex=r'.+\n\s(?P<line>\d+):(?P<column>\d+)'
+                     r'\s+✖\s(?P<message>.+)')
```

Comment:
@dracarys09  IMO this regex will not cover every message, they also have a warning symbol in place of cross. Can you check ?
and tests are not sufficient.